flyway-community-db-support/flyway-database-clickhouse/src/main/java/org/flywaydb/community/database/clickhouse/ClickHouseConfigurationExtension.java

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright (C) Red Gate Software Ltd 2010-2024
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.flywaydb.community.database.clickhouse;
+
+import lombok.Getter;
+import org.flywaydb.core.extensibility.ConfigurationExtension;
+
+import java.util.Map;
+
+@Getter
+public class ClickHouseConfigurationExtension implements ConfigurationExtension {
+    private static final String CLUSTER_NAME = "flyway.clickhouse.clusterName";
+    private static final String ZOOKEEPER_PATH = "flyway.clickhouse.zookeperPath";
```

Comment:
@sazonov I have noticed what I think is a typo, can this be confirmed?
Is `flyway.clickhouse.zookeperPath` meant to be `flyway.clickhouse.zookeeperPath`?

Reply:
Confirmed, that's a typo. Fixed that.

Reply:
Thanks @sazonov That was going to be a follow up as I saw that not being used.